Makefile

```diff
@@ -10,16 +10,16 @@ test:
 bench:
 	go test -v ./... -bench=. -timeout 2m
 
-run:
-	cd vflow; go run $(GOFILES) -sflow-workers 100 -ipfix-workers 100
```

Comment:
Once again with the go tool chain in the way things like +build windows does not line up here.  

I see a few choices:
* build then run 
* go run static list of files in makefile
* go run dynamic list of files in makefile